pages.bn/android/bugreportz.md

```diff
@@ -0,0 +1,21 @@
+# bugreportz
+
+> একটি জিপ করা অ্যান্ড্রয়েড বাগ রিপোর্ট তৈরি করুন।
+> এই কমান্ডটি শুধুমাত্র `adb shell` এর মাধ্যমে ব্যবহার করা যেতে পারে।
+> আরও তথ্য পাবেন: <https://cs.android.com/android/platform/superproject/+/master:frameworks/native/cmds/bugreportz>.
+
+- একটি অ্যান্ড্রয়েড ডিভাইসের একটি সম্পূর্ণ জিপ করা বাগ রিপোর্ট তৈরি করুন:
+
+`bugreportz`
+
+- চলমান `bugreportz` অপারেশনের অগ্রগতি দেখান:
+
+`bugreportz -p`
```

Comment:
Optional fix (error source: original page): `Show the progress of a running operation:` can be used as a translation source, `bugreport` is already mentioned in the page title, no need to duplicate it's name again.
Warning: no `Bengali` language available in `Google Translate`, suggestion unavailable.

Reply:
> Warning: no `Bengali` language available in `Google Translate`, suggestion unavailable.

Wdym, Bengali is available in Google translate. I will review your suggestions soon.

Reply:
![image](https://user-images.githubusercontent.com/42812113/210161102-e6fcd5f8-df95-4c58-97b9-c61f0a329b3b.png)

🤔

Reply:
> ![image](https://user-images.githubusercontent.com/42812113/210161102-e6fcd5f8-df95-4c58-97b9-c61f0a329b3b.png)
> 
> 🤔 Somebody or smth is mistaken

Type Bengali to English or something like that.

This is a direct link https://translate.google.com/?sl=bn&tl=en&op=translate